packages/api/src/cardano/api/cardano-api-maker.ts

```diff
@@ -0,0 +1,17 @@
+import {Fetcher, fetcher} from '@yoroi/common'
+
+import {getProtocolParams as getProtocolParamsWrapper} from './protocol-params'
+import {CardanoApi} from '../../index'
+export const cardanoApiMaker = ({
+  baseUrl,
```

Comment:
@banklesss update to receive the network `mainnet` | `preprod` | `sanchonet`, also create a `API_ENDPOINTS` config, i.e -> governance.